doc/tech/roadmap.md

```diff
@@ -46,7 +48,17 @@ A Release is:
 Please refer to individual [release plan](./release_notes/) and features for detailed roadmap.
 
 <a name="2.3"></a>
-### Milestones
+### Detailed Milestones
+| Review | Milestone | Description | Activities | Comments |
+|--------|-----------|--------------|------------|---------|
+| Kick-Off | M0 | The goal of the release Kick-Off is to open the innovation platform for the intent to participate in the CNTT release. Release Kick-Off review takes place for each releases. | Identity a list of features and functionality will be documented and developed. | |
+| Planning | M1 | The goal of the Release Planning review is to ensure the plan is complete, sufficient, and aligned with release milestones. All people resources are identified, documented and committed. | Identify what is in or out of scope. During the development phase, it helps reduce the discussion. Highlight the issues to TSC. | |
+| Issue Logging Freeze | M2 | The goal of the Issue Logging Freeze is to mark the end of adding new issues in the Release. |  All the project leads authorise the issues are correctly labelled, mapped to the corresponding project and milestone, etc., | |
```

Comment:
Instead of "Issue logging" can we use "Change Freeze" where requested changes are logged into Github as issues; I think post-M2 issues related to erros are still allowed.

Reply:
@pgoyal01  - I will better reword it to "Github Issue Logging" rather than "Issue Logging" or "Change Freeze". Again question might raise “what is change freeze”. The issues are logged not only to change, also to add, remove